src/labels/CustomLabels.labels

```diff
@@ -2971,6 +2971,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>campaignMemberStatusResponded</shortDescription>
         <value>campaignMemberStatusResponded</value>
     </labels>
+    <labels>
+        <fullName>CallableApiMethodNotImplemented</fullName>
+        <categories>API</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>CallableApi Method Not Implemented</shortDescription>
+        <value>''{0}'' is not implemented</value>
```

Comment:
@patrick-yan-sf  The double single quotes are not needed here. It could also be added as `&apos;{0}&apos;` to escape the single quotes